website_cookie_notice/__manifest__.py

```diff
@@ -9,6 +9,7 @@
     'category': 'Website',
     'author': "Agile Business Group, "
               "Tecnativa, "
+              "Callino"
```

Comment:
I'm very thankful for you contribution and improvement of this module, but I'm not sure if is fair to add you as an author after 2 lines of code. Usually contributor it is.